src/app/app.module.ts

```diff
@@ -8,6 +8,12 @@ import { ReactiveFormsModule } from '@angular/forms';
 import { HomeComponent } from './home/home.component';
 import { MaterialModule } from './material/material.module';
 import { SharedModule } from './shared/shared.module';
+import { LayoutModule } from '@angular/cdk/layout';
+import { MatToolbarModule } from '@angular/material/toolbar';
```

Comment:
why are we importing mat modules in root module? We already have a separate material module, please add dependent mat modules to the material module.

Reply:
I  removed mat-modules in the app-module.ts.